packages/react-widgets/src/DateTimePickerInput.js

```diff
@@ -62,16 +64,16 @@ class DateTimePickerInput extends React.Component {
   }
 
   render() {
-    let { disabled, readOnly } = this.props
+    let { className = '', disabled, readOnly, type = 'text' } = this.props
     let { textValue } = this.state
 
     let props = Props.omitOwn(this)
 
     return (
       <Input
         {...props}
-        type="text"
-        className="rw-widget-input"
+        type={type}
+        className={`rw-widget-input ${className}`}
```

Comment:
lets use `classnames` package for this like we do elsewhere to handle the null case